lib/Parser.js

```diff
@@ -632,24 +730,28 @@ Parser.prototype = {
         var filter = {
             type: 'text'
         };
-        if (rule.child !== undefined) {
+        if (rule.child) {
             filter.type = 'child';
             filter.value = rule.child;
-        }
-        else if (rule.attr !== undefined) {
+        } else if (rule.attr) {
             filter.type = 'attr';
             filter.value = rule.attr;
+        } else if (rule.prop) {
+            filter.type = 'prop';
+            filter.value = rule.prop;
         }
         debug('._parseSimpleRule() has called for selector %s with filter %o', selector, filter);
         return this._env.evaluateJs(selector, filter, /* @covignore */ function(selector, filter) {
             var nodes = Sizzle(selector);
             return nodes.map(function(node) {
                 switch (filter.type) {
-                    case 'attr':
-                        return node.getAttribute(filter.value);
                     case 'child':
                         var childNode = node.childNodes[filter.value];
                         return childNode.textContent;
+                    case 'attr':
+                        return node.getAttribute(filter.value);
+                    case 'prop':
```

Comment:
could you explain me one more time what is the difference between attribute and property?